src/mqueryfront/src/App.css

```diff
@@ -4,10 +4,26 @@
 }
 
 .mquery-scroll-matches {
-    height: calc(100vh - 95px);
+    height: calc(100vh - 235px);
     overflow-y: scroll;
 }
 
 .mquery-scroll-matches td {
     word-break: break-all;
 }
+
+#query-column {
```

Comment:
Is it possible to achieve this effect with css classes instead of IDs? AFAIK classes are strongly preferred over IDs in reactjs when possible. I *think* in this case simple class will be enough?